langstream-e2e-tests/src/test/java/ai/langstream/tests/util/kafka/RemoteKafkaProvider.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ai.langstream.tests.util.kafka;
+
+import ai.langstream.api.model.StreamingCluster;
+import ai.langstream.tests.util.BaseEndToEndTest;
+import ai.langstream.tests.util.StreamingClusterProvider;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.admin.Admin;
+
+@Slf4j
+public class RemoteKafkaProvider implements StreamingClusterProvider {
+
+    private static final String SYS_PROPERTIES_PREFIX = "langstream.tests.remotekafka.props.";
+    private static final Map<String, Object> ADMIN_CONFIG;
+
+    private org.apache.kafka.clients.admin.Admin admin;
```

Comment:
are we adding the langstream-kafka-runtime only for this class ?
it is better to add only the kafka-clients jar, and not import all the other dependencies